#Independent settings for the planner's costmap
visual_servoing:
  Kp_gain: 0.1
  Ki_gain: 0.1
  Kd_gain: 0.0
  upper_limit: 2.0
  lower_limit: -2.0
  sampling_time: 10.0
  scaling_factor: 1.5
